riptable/rt_dataset.py

```diff
@@ -2548,11 +2548,11 @@ def _to_unicode_if_string(arr):
                 categories = _to_unicode_if_string(col.category_array) if unicode else col.category_array
                 data[key]: pd.Categorical = pd.Categorical.from_codes(codes, categories=categories)
             elif isinstance(col, TypeRegister.DateTimeNano):
-                ccol = col.copy()
-                arr = ccol._timezone.fix_dst(ccol._fa)
-                arr = arr.astype('datetime64[ns]')
+                pd_datetime = pd.to_datetime(col)

```

Comment:
I think this performance of this line is potentially greatly improved by using `pd.DatetimeIndex` instead of `to_datetime`. 
![image](https://user-images.githubusercontent.com/9717992/131340867-a7e77afd-724a-41e2-bac6-5d6edde0881a.png)

Reply:
Good point. I found that `np.asarray` further decrease the time to microsecond level. The code is updated. Thanks.
FYI, I hit a bug when I just use the DateTimeNano array in the DatetimeIndex constructor. I'm not sure if it's a known issue but I don't want it to block this progress so I will use the `np.asarray` to make it a numpy array instead.
```
In [12]: df['a']
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-0030cc624053> in <module>
----> 1 df['a']

~/miniconda3/envs/riptable/lib/python3.8/site-packages/pandas/core/frame.py in __getitem__(self, key)
   3425                 if isinstance(self.columns, MultiIndex):
   3426                     return self._getitem_multilevel(key)
-> 3427                 return self._get_item_cache(key)
   3428
   3429         # Do we have a slicer (on rows)?

~/miniconda3/envs/riptable/lib/python3.8/site-packages/pandas/core/frame.py in _get_item_cache(self, item)
   3878
   3879             loc = self.columns.get_loc(item)
-> 3880             values = self._mgr.iget(loc)
   3881             res = self._box_col_values(values, loc).__finalize__(self)
   3882

~/miniconda3/envs/riptable/lib/python3.8/site-packages/pandas/core/internals/managers.py in iget(self, i)
    980         """
    981         block = self.blocks[self.blknos[i]]
--> 982         values = block.iget(self.blklocs[i])
    983
    984         # shortcut for select a single-dim from a 2-dim BM

~/miniconda3/envs/riptable/lib/python3.8/site-packages/pandas/core/internals/blocks.py in iget(self, i)
    348
    349     def iget(self, i):
--> 350         return self.values[i]
    351
    352     def set_inplace(self, locs, values):

~/miniconda3/envs/riptable/lib/python3.8/site-packages/pandas/core/arrays/datetimelike.py in __getitem__(self, key)
    320         only handle list-likes, slices, and integer scalars
    321         """
--> 322         result = super().__getitem__(key)
    323         if lib.is_scalar(result):
    324             return result

~/miniconda3/envs/riptable/lib/python3.8/site-packages/pandas/core/arrays/_mixins.py in __getitem__(self, key)
    192         if lib.is_integer(key):
    193             # fast-path
--> 194             result = self._ndarray[key]
    195             if self.ndim == 1:
    196                 return self._box_func(result)

~/Projects/riptable/codes/riptable/rt_datetime.py in __getitem__(self, fld)
    644             # if result.strides[0] != result.itemsize:
    645             #    result = result.copy()
--> 646             result = self.newclassfrominstance(result, self)
    647         if np.isscalar(result):
    648             return self.get_scalar(result)

~/Projects/riptable/codes/riptable/rt_datetime.py in newclassfrominstance(cls, instance, origin)
   3840         '''
   3841         result = instance.view(cls)
-> 3842         result._timezone = origin._timezone.copy()
   3843         result._display_length = origin._display_length
   3844

AttributeError: 'DateTimeNano' object has no attribute '_timezone'

```

Reply:
Yes, I saw this as well, when doing ``col.view(int)`, which is why I used `.astype` to make a copy.